blockchain/types/src/block_metadata.rs

```diff
@@ -22,9 +47,10 @@ pub struct BlockMetadataContents {
     #[prost(message, required, tag = 2)]
     quorum_set: QuorumSet,
 
-    /// IAS report for the enclave which generated the signature.
-    #[prost(message, required, tag = 3)]
-    verification_report: VerificationReport,
+    /// Attestation evidence for the enclave which generated the signature.
+    #[prost(oneof = "AttestationEvidence", tags = "3, 5")]
+    #[digestible(name = "verification_report")]
+    attestation_evidence: Option<AttestationEvidence>,
```

Comment:
I think we need to make sure this is the path we want to go.

Another option is to create a new  enum that contains `BlockMetadataContents` and `BlockMetadataContentsDcap` or similar and then during decode we key off of the block version for which variant to decode.

Reply:
I like this better, but it needs to be tested carefully. We must make sure that the digestible output of an existing BlockMetadataContents remains the same.

EDIT: saw the test for this, nice!